### ISSUE TITLE
Document the qfpxfd algorithm

### ISSUE THREAD

**Issue by reubano** [concept-to-clinic/concept-to-clinic#22]
## Overview

Participants in the [Data Science Bowl](https://www.kaggle.com/c/data-science-bowl-2017) produced several algorithms that we would like to incorporate. To help facilitate this effort, we also want to add documentation so that contributors can make an educated decision when selecting an algorithm to incorporate.

## Expected Behavior

This documentation should enable people to:
 - view the library dependencies and license
 - understand its pros/cons
 - evaluate its performance/accuracy
 - identify which areas of the codebase to target for improvement

Design doc reference: [Detect and select](https://concept-to-clinic.readthedocs.io/en/latest/design-doc.html#detect-and-select)

## Algorithm info

key | value
----|------
team|qfpxfd
rank|4
repo|http://www.cis.pku.edu.cn/faculty/vision/wangliwei/software.html
trained models|not available
converted branch|
ML engine|Keras
engine-version|
ML backend|Tensorflow
backend-version|
training method|CNN
architecture|3D VGG
algorithm|
OS|
OS version|
Python version|
CUDA version|
cuDNN version|
notes|

## Technical details

 - This feature should be implemented as a new markdown file in the [`docs folder`](https://github.com/concept-to-clinic/concept-to-clinic/tree/master/docs)

## Acceptance criteria

 - [ ] effective documentation for the above

*NOTE: All PRs must follow the standard PR checklist.*

**Comment by vessemer** [comment]
Seems that [repo-link](www.cis.pku.edu.cn/faculty/vision/wangliwei/software.html) became outdated and quick search doesn't provide the answer for me.

**Comment by lamby** [comment]
Merged :)
